tools/bwa.wdl

```diff
@@ -209,6 +209,10 @@ task bwa_mem {
             description: "Read group information for BWA to insert into the header. BWA format: '@RG\tID:foo\tSM:bar'",
             group: "common",
         }
+        skip_mate_rescue: "Skip mate rescue. By default, if one read maps and the other does not, `bwa` will perform a Smith-Waterman alignment to attempt to 'rescue' the mate and produce an alignment. For certain data types (e.g. Hi-C), it may be desirable to disable mate rescue as the mates are not expected to conform to assumptions."
+        skip_pairing: "Skip pairing; mate rescue performed unless `skip_mate_rescue` also in use. With `skip_pairing` enabled, `bwa` will allow hits for paired data that do not fit the definition of a proper pair."
+        split_smallest: "For split alignment, take the alignment with the smallest coordinate as primary"
+        short_secondary: "Mark shorter split hits as secondary"
```

Comment:
Similar Q. What's the alternative here?

Reply:
`Mark shorter split hits as secondary (for Picard compatibility).`

It is apparently for compatibility with Picard. Other than that, the manual has no details. Google returns a biostars thread with the following:

```
without -M, a split read is flagged as 2048 (supplementary alignment) see http://picard.sourceforge.net/explain-flags.html. This flag is a recent addition to the SAM spec.
with option -M it is flagged as a duplicate flag=256 (not primary alignment): will be ignored by most 'old' tools.
```

Reply:
Though that answer conflicts with the description. They claim it is a `duplicate`, but the manual labels it `secondary`.